### DISCUSSION TITLE
Selection without drag and drop?

### DISCUSSION THREAD

**Discussion by surajravalhear** [ThibaultJanBeyer/DragSelect#79] [Q&A] [answered]
Is it possible to disable dragging and dropping elements, while still allowing drag selection? I need my elements to be fixed, but I need to toggle selection based on click / drag.

**Comment by ThibaultJanBeyer** [comment, marked as answer]
Dear @surajravalhear, thanks a lot for using the tool!

Sure, check the docs, there is `draggability`. So `draggability: false`. And prob. also `keyboardDrag: false`.

Was the description not helpful enough? Would you mind pointing me towards what was missing so you did not find it?
Thank you! Cheers :)

**Reply by ThibaultJanBeyer** [reply]
@arleybls you reacted with "thumbs down", would you mind elaborating why? Thank you!
